### DISCUSSION TITLE
Unable to upgrade with education benefits

### DISCUSSION THREAD

**Discussion by annaliw** [community/community#63966] [GitHub Education] [answered]
### Hi everyone,

I followed the instructions found in this discussion to upgrade my organization using my education benefits, but the upgrade is not showing up. I'm still listed as being on the free plan, even though I upgraded in the education dashboard. 

https://github.com/orgs/community/discussions/50702

**Comment by othman922** [comment]
#

**Comment by advayc** [comment, marked as answer]
The education benefits should show up a couple days after you send the request. Someone has to accept the request so just be patient.

**Reply by annaliw** [reply]
I see, thanks so much for the info!

**Reply by annaliw** [reply]
It took about a week for the benefits to show up. Thanks!

**Reply by mattiacentonze** [reply]
Hi all, I request the education benefits to get Copilot free but the request gives me an error: "Unable to submit your application. Please try again"

**Reply by Gyanaa-Vaibhav** [reply]
Are you able to use it now or is issue still same

**Comment by mattiacentonze** [comment]
Hi,

I already got the upgrade dome weeks ago.

Thank you!

MC

-------- Original Message --------
